src/project/module.ts

```diff
@@ -3,12 +3,31 @@ import { glob, tjs } from "./deps.ts";
 import { configPath as moduleConfigPath, readConfig as readModuleConfig } from "../config/module.ts";
 import { ModuleConfig } from "../config/module.ts";
 import { Script } from "./script.ts";
+import { ApiRoute } from "./api.ts";
 import { Project } from "./project.ts";
 import { Registry } from "./registry.ts";
 import { validateIdentifier } from "../types/identifiers/mod.ts";
 import { Casing } from "../types/identifiers/defs.ts";
-import { ProjectModuleConfig } from "../config/project.ts";
+import { configPath as projectConfigPath, ProjectModuleConfig } from "../config/project.ts";
 import { UserError } from "../error/mod.ts";
+import { ApiRouteConfig } from "../config/module.ts";
+
+function validateAndCleanPath(path: string, configPath: string): string {
```

Comment:
the path shouldn't be cleaned. it should be invalid and throw an error if there's something wrong with it.

Reply:
I need to look into how `resolve` works, and if not removing slashes is okay.

Reply:
So the reason I'm cleaning it is that the `resolve` function remove the base path if the subpath starts with a slash.

Reply:
The validation is performed first— I believe that we agreed that trailing slashes should have no semantic difference, so that would need to be cleaned either way

Reply:
> I believe that we agreed that trailing slashes should have no semantic difference

ah, that's where the misunderstanding is. don't remember this convo, but i believe we should force a specific style for consistency.

Reply:
Done!